src/faction_camp.cpp

```diff
@@ -289,14 +289,14 @@ void talk_function::become_overseer( npc &p )
     p.mission = NPC_MISSION_GUARD_ALLY;
     p.chatbin.first_topic = "TALK_CAMP_OVERSEER";
     p.set_destination();
-    get_basecamp( p );
+    get_basecamp( p.global_omt_location() );
 }
 
 
-void talk_function::camp_missions( mission_data &mission_key, npc &p )
+void basecamp::camp_missions( mission_data &mission_key, npc &p )
 {
     std::string entry;
-    basecamp *bcp = get_basecamp( p );
+    basecamp *bcp = get_basecamp( p.global_omt_location() );
```

Comment:
If we don't have a basecamp at this point, we are severely hosed, because this is a basecamp member function.